components/ui/scroll-throttled.tsx

```diff
@@ -0,0 +1,33 @@
+import { useState, useEffect } from "react";
+import { throttle } from "@martinstark/throttle-ts";
+// import { debounce } from "ts-debounce";
+
+function useDocumentScrollThrottled(callback: {
+  (callbackData: { previousScrollTop: Number; currentScrollTop: Number }): void;
+  (arg0: { previousScrollTop: number; currentScrollTop: number }): void;
+}) {
+  const [, setScrollPosition] = useState(0);
```

Comment:
What's the point of setting the state here if you're not using the value at all?

Reply:
@zainfathoni I've deleted this file and merged the function to `global-header.tsx` please check it again mas